spec/factories/ems_cluster.rb

```diff
@@ -9,5 +9,29 @@
     end
   end
 
+  trait :vmware_ems do
```

Comment:
I don't see much benefit of adding a trait that creates a new ems and sets it, when you could just do:
```
let(:ems_vmware) { FactoryBot.create(:ems_vmware) }
let(:vmware_cluster) { FactoryBot.create(:ems_cluster, :ext_management_system => ems_vmware) }
```

Especially because that more easily allows you to create multiple clusters on the same ems

Reply:
It was added when I was being introduced by a help.  Do I need to take them out?

Reply:
Please, it is extra code which I don't see being very useful anywhere else

Reply:
removed the traits and refactored the affected code.